### PR TITLE
Move the introduction page from SIG CLI guides to the kubectl reference

### LINKED ISSUE #45600: Move the introduction page from SIG CLI guides to the kubectl reference
**This is a Feature Request**

**What would you like to be added**

Move the `Introduction` page from the [SIG CLI guides](https://kubectl.docs.kubernetes.io/guides) to the [upstream kubectl reference](https://kubernetes.io/docs/reference/kubectl)

**Why is this needed**

Multiple versions of the kubectl documentation exists which can be merged into the kubectl reference section in the Kubernetes upstream website. Part of https://github.com/kubernetes/website/issues/25943

**Comments**

/good-first-issue
/language-en
/help-wanted

### PR DESCRIPTION
**Pull request by prashantrewar**
Fixes #45600

### CODE REVIEW

File: content/en/docs/reference/kubectl/Introduction/kubectl.md
Comment (sftim) [inline content/en/docs/reference/kubectl/Introduction/kubectl.md:1]: I recommend using the path `content/en/docs/reference/kubectl/introduction.md`

File: content/en/docs/reference/kubectl/Introduction/kubectl.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+description: >
+    Introduction to Kubectl
+---
+
+{{< alert color="success" title="TL;DR" >}}
+- Kubectl is the Kubernetes cli
+- Kubectl provides a swiss army knife of functionality for working with Kubernetes clusters
+- Kubectl may be used to deploy and manage applications on Kubernetes
+- Kubectl may be used for scripting and building higher-level frameworks
+{{< /alert >}}
```
Comment (sftim) [inline content/en/docs/reference/kubectl/Introduction/kubectl.md:15]: Thanks. Try this:
```suggestion
---
title: "Introduction To Kubectl"
content_type: concept
weight: 1
---
```

File: content/en/docs/reference/kubectl/Introduction/kustomize.md
Comment (sftim) [inline content/en/docs/reference/kubectl/Introduction/kustomize.md:1]: Please don't include this page. Another PR could try moving it over.

**Review by sftim — COMMENTED** [review]
Thanks. I think you could simplify this PR to move just one page over.

Sorry that the issue wasn't clear about what to do.

File: content/en/docs/reference/kubectl/Introduction/kustomize.md
Comment (prashantrewar) [reply content/en/docs/reference/kubectl/Introduction/kustomize.md:1]: Sure!

File: content/en/docs/reference/kubectl/Introduction/kubectl.md (hunk 1 above)
Comment (prashantrewar) [reply content/en/docs/reference/kubectl/Introduction/kubectl.md]: Done!

**Comment by sftim** [comment]
LGTM

What do other contributors think of this proposed move?

**Comment by stmcginnis** [comment]
Makes sense!

/lgtm

**Comment by sftim** [comment]
/approve

Let's do it.

**Pull request merged**